containers/images/pulp-resource-manager/container-assets/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+/usr/bin/wait_on_database_migration.py
```

Comment:
what are the chances of a race condition if you run multiple pods?

Reply:
Race condition for what to happen? We run multiple pulp-workers which has this same wait without issue.